src/OmniSharp.Script/ScriptHelper.cs

```diff
@@ -88,8 +94,9 @@ private CachingScriptMetadataResolver CreateMetadataReferenceResolver()
                 }
             }
             
-            return enableScriptNuGetReferences ? new CachingScriptMetadataResolver(new NuGetMetadataReferenceResolver(ScriptMetadataResolver.Default)) 
-                : new CachingScriptMetadataResolver(ScriptMetadataResolver.Default);
+            return enableScriptNuGetReferences ?
+                new CachingScriptMetadataResolver(new NuGetMetadataReferenceResolver(_resolver)) :
+                new CachingScriptMetadataResolver(_resolver);
```

Comment:
nit: for this conditional expression, you have the `?` and `:` at the end of the lines. However, below, you have them at the start of the lines. Maybe make these consistent? (I personally prefer the bottom one.)